db/migrate/20190814163542_create_messages.rb

```diff
@@ -0,0 +1,22 @@
+class CreateMessages < ActiveRecord::Migration[5.2]
+  def change
+    create_table :messages, id: :uuid, default: "uuid_generate_v4()" do |t|
+      t.text :sender_message_id
+      t.text :title, null: false
+      t.text :path
```

Comment:
Looks like this needs to be changed to url

Reply:
Good point, I've amended that now.